Dockerfile

```diff
@@ -1,22 +1,49 @@
-FROM tomcat:8.5-jdk8-openjdk
-MAINTAINER geosolutions<info@geo-solutions.it>
+FROM alpine AS mother
+LABEL maintainer="Alessandro Parma<alessandro.parma@geo-solutions.it>"
+ARG MAPSTORE_WEBAPP_SRC="./.placeholder"
+ADD "${MAPSTORE_WEBAPP_SRC}" "/mapstore/"
+
+COPY ./docker/* /mapstore/docker/
+WORKDIR /mapstore
+
+FROM tomcat:9-jdk11-openjdk
 
 # Tomcat specific options
 ENV CATALINA_BASE "$CATALINA_HOME"
-ENV JAVA_OPTS="${JAVA_OPTS}  -Xms512m -Xmx512m -XX:MaxPermSize=128m"
+ENV MAPSTORE_WEBAPP_DST="${CATALINA_BASE}/webapps"
+ENV INITIAL_MEMORY="512m"
+ENV MAXIMUM_MEMORY="512m"
+ENV JAVA_OPTS="${JAVA_OPTS}  -Xms${INITIAL_MEMORY} -Xmx${MAXIMUM_MEMORY} -XX:MaxPermSize=128m"
+ENV GEOSTORE_OVR_OPT="-Dgeostore-ovr=${CATALINA_BASE}/conf/${OVR} \
+                      -Duser.timezone=UTC"
+ARG TOMCAT_EXTRAS=false
+ARG OVR=""
 
 # Optionally remove Tomcat manager, docs, and examples
-ARG TOMCAT_EXTRAS=false
 RUN if [ "$TOMCAT_EXTRAS" = false ]; then \
```

Comment:
I think the whole "remove TOMCAT_EXTRAS" stuff is not needed anymore with the new tomcat base image:
```
docker run -it tomcat:9-jdk11-openjdk bash
Unable to find image 'tomcat:9-jdk11-openjdk' locally
9-jdk11-openjdk: Pulling from library/tomcat
b9a857cbf04d: Already exists
d557ee20540b: Already exists
3b9ca4f00c2e: Already exists
667fd949ed93: Already exists
b6a945768a17: Already exists
aa2f45feafbf: Already exists
7179e0e40c20: Already exists
d659cf2d36fd: Already exists
880a7c791192: Pull complete
dd17efe07475: Pull complete
Digest: sha256:e7bc48bd0b0c18db84e718769ac27c87f90f9ecb20b761700be0a016a7f840d2
Status: Downloaded newer image for tomcat:9-jdk11-openjdk
root@8ba65a0295ff:/usr/local/tomcat# ls webapps

```

the webapps directory is empty already in the base image itself

Reply:
fixed in this commit https://github.com/geosolutions-it/MapStore2/pull/6490/commits/81f252222042c06992b67e8602977de593b31564